### PR TITLE
update sh/python/ruby/cpp.snip

### PR DESCRIPTION
**Pull request by e-kwsm**

### CODE REVIEW

File: neosnippets/c.snip [hunk 1]
```diff
@@ -24,8 +24,8 @@ abbr        if () {} else {}
 
 snippet     for
 abbr        for () {}
-    for (${1} = 0; $1 < ${2}; $1++) {
-        ${0:TARGET}
+    for (${1:int} ${2:i} = ${3:0}; $2 < ${4}; $2++) {
+        ${0:#:TARGET}
```
Comment (ujihisa) [inline neosnippets/c.snip]: It's not valid for non C99

**Comment by ujihisa** [comment]
This pull request includes lots of commits which are reverted. Can you squash them?

**Comment by e-kwsm** [comment]
>It's not valid for non C99

You can delete type specifier and jump to ```$2```, as non C99.

>This pull request includes lots of commits which are reverted. Can you squash them?

I squashed, and further pulled request: I'm sorry if this confuses you.

**Comment by Shougo** [comment]
OK.

**Pull request merged**